content/influxdb/v2.0/reference/internals/system-buckets.md

```diff
@@ -0,0 +1,84 @@
+---
+title: InfluxDB system buckets
+description: >
+  InfluxDB system buckets contain time series data used by and generated from the
+  InfluxDB monitoring and alerting system and the task engine.
+menu:
+  influxdb_2_0_ref:
+    name: System buckets
+    parent: InfluxDB internals
+weight: 101
+influxdb/v2.0/tags: [buckets]
+related:
+  - /influxdb/v2.0/monitor-alert/
+  - /influxdb/v2.0/process-data/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/check/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/deadman/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/from/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/logs/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/notify/
+---
+
+InfluxDB system buckets contain time series data used by and generated from the
+InfluxDB monitoring and alerting system and the task engine.
+
+The following system buckets exist in each InfluxDB organization:
+
+- [_monitoring](#_monitoring-system-bucket)
+- [_tasks](#_tasks-system-bucket)
+
+## _monitoring system bucket
+The `_monitoring` system bucket stores data generated from and used by InfluxDB
+[checks and notifications](/influxdb/v2.0/monitor-alert/).
+
+### _monitoring bucket schema
+
+- **statuses** _(measurement)_
+  - **tags:**
+    - **\_check\_id:** check ID
+    - **\_check\_name:** check name
+    - **\_level:** level evaluated by the check (ok, info, warn, or crit)
+    - **\_source\_measurement:** original measurement queried by the check
+    - **\_type:** check type (threshold or deadman)
+    - _tags inherited from queried data_
+  - **fields:**
+    - **\_message:** message generated by the check
+    - **\_source_timestamp:** original timestamp of the queried data
+    - _other fields inherited from queried data_
+- **notifications** _(measurement)_
+  - **tags:**
+    - **\_check\_id:** check ID that triggered the notification
+    - **\_check\_name:** check name that triggered the notification
+    - **\_level:** check-evaluated level that triggered the notification (ok, info, warn, or crit)
+    - **\_notification_endpoint_id:** notification endpoint ID
+    - **\_notification_endpoint_name:** notification endpoint name
+    - **\_notification_rule_id:** notification rule ID
+    - **\_notification_rule_name:** notification rule name
+    - **\_sent:** sent status (true or false)
+    - **\_source\_measurement:** original measurement queried by the check
+    - **\_type:** check type (threshold or deadman)
+    - _tags inherited from queried data_
+  - **fields:**
+    - **\_message:** message generated by the check
+    - **\_source_timestamp:** original timestamp of the queried data
+    - **\_status_timestamp:** timestamp when the status (`_level`) was evaluated
+    - _other fields inherited from queried data_
+
+## _tasks system bucket
+The `_tasks` system bucket stores data related to [InfluxDB task](/influxdb/v2.0/process-data/) executions.
```

Comment:
used to process [InfluxDB tasks]...that you've created?

Reply:
This data isn't used to process tasks. It's data created when a task runs.